test_integration/test_ee_short_region_slop.cpp

```diff
@@ -53,11 +53,11 @@ int main(int argc, char **argv)
     double duration = 0.2048;
     double repeat = 200; // Each trial takes 41 seconds at sticker frequency
     for (size_t duration_idx = 0; duration_idx != num_duration; ++duration_idx) {
-        // Create scaling and scaling_timed model regions
```

Comment:
Is this reversal required for performance?  If so I'd like to understand why, and if not, we should revert this part of the change.